xAPI.md

```diff
@@ -610,19 +610,17 @@ field is the [Medbiquitous xAPI Interest Group](http://groups.medbiq.org/medbiq/
 
 ###### Description
 
-Communities of practice will, at some point in time, need to establish new Verbs to meet the needs of their constituency.
+Communities of practice could find the need to establish new Verbs, Activities, Extensions, etc. to meet the needs of their constituency.
 
 Therefore, it is expected that xAPI communities of practice generate profiles, lists, and repositories that become 
-centered on Verb vocabularies.  ADL is creating a companion document containing Verbs for xAPI to serve the ADL Community.
-
-In fulfillment of the requirements below, a collection of IRIs of recommended Verbs exists.  There are times when 
-Activity Providers might wish to use a different Verb for the same meaning.
+centered on vocabularies.  Verbs are likely to be the most common of these vocabularies, as many profiles begin by defining
+them.  There are times when Activity Providers might wish to use a different Verb for the same meaning.
```

Comment:
> There are times when Activity Providers might wish to use a different Verb for the same meaning.

This line has been around for a while, and wasn't changed here, but it still seems like a reasonable venue to discuss it. 

Saying _similar_ meaning might make this more accurate. For example, even if cmi5 were to coin a `"...completed"` verb (it doesn't), it would necessarily have additional context by being a verb defined for the cmi5 spec -- one could infer the slightly different meaning `"Completed a cmi5 AU"` vs `"Completed a learning activity"` for the ADL `"...completed"` verb.

Even better from my point of view would be to remove this line. Note the direction we actually went with cmi5, we used the existing ADL `"...completed"` verb rather than coining a new one, because we decided it was better to avoid coining new verbs for such similar meanings, and rather add the context that this is a cmi5 statement using [contextActivities] (https://github.com/adlnet/xAPI-Spec/blob/master/xAPI.md#4162-contextactivities-property). That seems like a better practice to encourage.

On a related note, again we've had this for a while, but I think we should avoid talking too much about "vocabularies" as they're not really a core xAPI concept (which only has requirements for individual identifiers), and "vocabulary" vs "mix and match" is an area of disagreement about best practice.